examples/component/wasm/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "example-component-wasm"
+version = "0.0.0"
+authors = ["The Wasmtime Project Developers"]
+edition = "2021"
+publish = false
+
+[dependencies]
+wit-bindgen = { workspace = true, default-features = false, features = ["macros"] }
```

Comment:
For this the `default-features = false` isn't probably the best example to give since default features should be ok for 99% of embeddings. The reason it's disabled in this repository is for the unique use case of the adapter.

Reply:
Fixed by using the default.